cluster/schema/meta_class.go

```diff
@@ -482,7 +483,15 @@ func (m *metaClass) applyShardProcess(name string, action command.TenantProcessR
 
 		if count == len(processes) {
 			copy.Status = req.Tenant.Status
-			// TODO move the data events channel out of cluster/schema and handle at a higher/different level
+			// TODO move the data version/data events channel out of cluster/schema and handle at a higher/different level
+
+			// modify the copy, which will be stored in m.Sharding.Physical by the caller
+			copy.DataVersion++
```

Comment:
what happens if multiple clients to offload the tenant at the same time? is it concurrent safe?